content/docs/guides/datadog.md

```diff
@@ -62,6 +64,446 @@ Additionally, if you are setting up Neon’s Datadog integration for a project w
 
 Once integrated, you can create custom dashboards in Datadog by querying the metrics sent from Neon. Use Datadog's **Metrics Explorer** to search for metrics like `connection_counts`, `db_total_size`, and `host_cpu_seconds_total`. You can also set alerts based on threshold values for critical metrics.
 
+## Import the Neon dashboard
+
+As part of the integration, Neon provides a JSON configuration file that you can import into Datadog to start with a pre-built dashboard based on Neon metrics.
+
+![neon dashboard in datadog](/docs/guides/neon-dashboard-datadog.png)
+
+Here's how you can import the dashboard:
+
+1. In the Neon Console, open your Datadog integration from the **Integrations** page.
+1. Scroll to the bottom of the panel and copy the JSON from there.
+
+   OR
+
+   You can copy the JSON below instead.
+
+    <details>
+    <summary>Neon dashboard</summary>
+    ```json shouldWrap
```

Comment:
I hope this look OK on the website as it doesn't look pretty in the Github diff interface :)

Reply:
I moved it out of the indent - hopefully it looks a bit better. For something this long, I'd rather have it as a download. We can explore that in the next update.